inbox/push-priority.xml

```diff
@@ -127,7 +127,7 @@
   </section2>
 </section1>
 <section1 topic='Design Considerations' anchor='design'>
-  <p>Alternatively to the approach described in this document, it was considered to determine priority of push notification based on existence of &xep0004; of 'FORM_TYPE' equal 'urn:xmpp:push:summary' within 'notification' element defined in &xep0357;. That approch however was not normative as &xep0357; suggests usage of this data form element, but only if summarized information is to be provided. That could lead to assuming that some of high priority notifications are actually a low priority once.</p>
+  <p>Alternatively to the approach described in this document, it was considered to determine priority of push notification based on existence of &xep0004; of 'FORM_TYPE' equal 'urn:xmpp:push:summary' within 'notification' element defined in &xep0357;. That approach however was not normative as &xep0357; suggests usage of this data form element, but only if summarized information is to be provided. That could lead to assuming that some of high priority notifications are actually a low priority once.</p>
```

Comment:
In above changes you replaced 'notification' with <notification/>. We should keep it in sync so that we would always refer to the element in the same way, shouldn't we?

Reply:
done